### PR TITLE
Generating Azure .NET SDK for Azure Large Instance API

### PR DESCRIPTION
**Pull request by 8Gitbrix**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

## Service Information
This PR adds the .NET sdk for the [Azure Large Instance](https://github.com/Azure/azure-rest-api-specs/blob/2299e0ee52b170edc9c7b50e1a864501d514692f/specification/azurelargeinstance/resource-manager/readme.md) service (more [info](https://learn.microsoft.com/en-us/azure/azure-large-instances/what-is-azure-large-instances) on the service)

## Testing Exception
Similar to the CLI extension, we'd like an exception on testing because we don't have the resources to dedicate a blade / machine for a test. We have to manually create resources for customers since it's not an ARM operation, and our other operations such as restarting an instance are intensive to be tested periodically. See teams post [here](https://teams.microsoft.com/l/message/19:7b87fb348f224b37b6206fa9d89a105b@thread.skype/1704312889404?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&parentMessageId=1704312889404&teamName=Azure%20SDK&channelName=Language%20-%20DotNet&createdTime=1704312889404)

### CODE REVIEW

**Review by jsquire — DISMISSED** [review]
@8Gitbrix :

You're missing "write" permissions to the repository.  Please review the Azure SDK onboarding documentation and request the correct set of permissions.

- [Azure SDK onboarding](https://eng.ms/docs/products/azure-developer-experience/onboard/requirements) _(Microsoft internal)_
- [Azure SDK onboarding assistance](https://teams.microsoft.com/l/channel/19%3ade3fce22c2994be18cac50502c55f717%40thread.skype/Azure%2520SDK%2520Onboarding?groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47) _(Microsoft internal)_

Please also be sure to add yourself to [CODEOWNERS](https://github.com/Azure/azure-sdk-for-net/blob/main/.github/CODEOWNERS) for this library, if you will be maintaining it going forward.

**Comment by 8Gitbrix** [comment]
@jsquire Thanks for letting me know. Sorry which Azure-sdk-write org am I supposed to join?

**Comment by jsquire** [comment]
> @jsquire Thanks for letting me know. Sorry which Azure-sdk-write org am I supposed to join? EDIT: NVM, I see there's an azure-sdk-write, just submitted a request to join

Please follow the linked onboarding guide, which walks you through requesting permissions.   Please use the linked Teams channel for assistance.

**Comment by 8Gitbrix** [comment]
Does anything else need to be done on my end to merge this?

**Comment by jsquire** [comment]
> Does anything else need to be done on my end to merge this?

You'll need to work with @ArthurMa1978 and team to have this reviewed.

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -860,3 +860,6 @@
 
 # PRLabel: %Spring App Discovery
 /sdk/springappdiscovery/Azure.ResourceManager.*/                    @sunkun99
+
+# Azure Large Instance SDK
+/sdk/azurelargeinstance                                             @8Gitbrix
```
Comment (jsquire) [inline .github/CODEOWNERS:865]:
```suggestion
# PRLabel: %Large Instance
/sdk/azurelargeinstance                                             @8Gitbrix

# ServiceLabel: %Large Instance %Service Attention
#/<NotInRepo>/                                                     @8Gitbrix
```

**Review by ArthurMa1978 — APPROVED** [review]

**Pull request merged**